Tribler/community/tunnel/routing.py

```diff
@@ -95,7 +96,14 @@ def tunnel_data(self, destination, payload):
         @return bool: whether the tunnel request has succeeded, this is in no
          way an acknowledgement of delivery!
         """
-        return self.proxy.tunnel_data_to_end(destination, payload, self)
+
+        self._logger.error("Tunnel data (len %d) to end for circuit %s with ultimate destination %s", len(payload),
```

Comment:
We should lower this, any level will be shown if a test fail and if you are debugging the tunnels locally, you can always edit logger.conf to lower this particular logger.